backend/nlp/handler.py

```diff
@@ -99,13 +112,32 @@ def analyze_reviews(reviews_list):
                 #if abs(sentiment_scores['compound']) > abs(compound):
                 compound += sentiment_scores['compound']
 
-            processed_review['neg'] = bad
-            processed_review['pos'] = good
-            processed_review['compound'] = compound
+            review['negSentiment'] = int(bad * 100)
+            review['posSentiment'] = int(good * 100)
```

Comment:
We also need to save the neutral sentiment.

Reply:
correct, i'll make sure to add that and have it ready for the morning